### PR TITLE
Work -> projects

### PR DESCRIPTION
**Pull request by JoaquinTrinanes**
This PR adds the `Featured projects` and `Other projects` sections to the new `/work` page

### CODE REVIEW

File: src/components/decoration/sprite/index.tsx [hunk 1]
```diff
@@ -49,19 +48,31 @@ const Sprite: React.FC<SpriteProps> = ({
 
   const initialPositionPx = isMdScreen ? 0 : pixelsLeft * pixelSize;
   const finalPositionPx = isMdScreen
-    ? width - image.width * scale
+    ? width - image.width * scale - 20
```
Comment (MatsThieme) [inline src/components/decoration/sprite/index.tsx:51]: I think we should just hide the animals on <md resolutions, as this is how our other pages handle it.
Also, isn't this the final position for md screens, not mobile?

File: src/components/decoration/carousel.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            ref={(ref) => {
+              listItemsRef.current[i] = ref;
+            }}
+          >
+            {item}
+          </li>
+        ))}
+      </ul>
+      <div className="flex flex-row gap-3 justify-center items-center flex-wrap">
+        {Array(pages)
```
Comment (MatsThieme) [inline src/components/decoration/carousel.tsx]: How about `[...new Array(10)].map`...?

**Review by MatsThieme — APPROVED** [review]
I didn't have the time to do an extensive review, but here is what I found.
grafik
One side in the carousel is cut off as shown in the screenshot.
That being said, it already looks great!

File: src/components/decoration/sprite/index.tsx (hunk 1 above)
Comment (JoaquinTrinanes) [reply src/components/decoration/sprite/index.tsx:51]: I'm not sure I follow, this is the component we use for all animated animals, so we're not hiding them on mobile? :thinking: 

Screens < md and mobile are almost always the same thing, and in most cases md is a good point to change the layout (sm might be too small)

Comment (MatsThieme) [reply src/components/decoration/sprite/index.tsx:51]: Sorry, I confused the hiding of blocks with the sprites.

Right, I assumed `isMdScreen` was >md :D

**Pull request merged**